index.js

```diff
@@ -1,44 +1,51 @@
 'use strict';
-
+require('dotenv-safe').load();
 const http = require('http');
 const express = require('express');
-const { urlencoded } = require('body-parser');
+const {urlencoded} = require('body-parser');
 const twilio = require('twilio');
+const ClientCapability = twilio.jwt.ClientCapability;
+const VoiceResponse = twilio.twiml.VoiceResponse;
+
 
-let app = express();
+const app = express();
 app.use('/node_modules', express.static(__dirname + '/node_modules'));
 app.use(express.static(__dirname + '/public'));
-app.use(urlencoded({ extended: false }));
+app.use(urlencoded({extended: false}));
 
 // Generate a Twilio Client capability token
 app.get('/token', (request, response) => {
-  let capability = new twilio.Capability(
-    process.env.TWILIO_ACCOUNT_SID,
-    process.env.TWILIO_AUTH_TOKEN
+  const capability = new ClientCapability({
+    accountSid: process.env.TWILIO_ACCOUNT_SID,
+    authToken: process.env.TWILIO_AUTH_TOKEN
+  });
+
+  capability.addScope(
+    new ClientCapability.OutgoingClientScope({
+      applicationSid: process.env.TWILIO_TWIML_APP_SID})
   );
-  capability.allowClientOutgoing(process.env.TWILIO_TWIML_APP_SID);
 
-  let token = capability.generate();
+  const token = capability.toJwt();
 
   // Include token in a JSON response
   response.send({
-    token: token
+    token: token,
```

Comment:
It's a matter of convention, in this came in the default settings of the linter. It's actually useful sometimes, as if you add a new element you won't forget to add the comma. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas